libs/agno/agno/tools/cartesia.py

```diff
@@ -0,0 +1,534 @@
+            language (Optional[str], optional): The language code. Defaults to None.
+            base_voice_id (Optional[str], optional): The ID of the base voice. Defaults to None.
+
+        Returns:
+            str: JSON string containing the created voice information.
+        """
+        logger.info(f"Creating voice: {name}")
+        try:
+            params = {
+                "name": name,
+                "description": description,
+                "embedding": embedding
+            }
+
+            if language:
+                params["language"] = language
+
+            if base_voice_id:
+                params["base_voice_id"] = base_voice_id
+
+            result = self.client.voices.create(**params)
+            return json.dumps(result, indent=4)
+        except Exception as e:
+            logger.error(f"Error creating voice with Cartesia: {e}")
+            return json.dumps({"error": str(e)})
+
+    def change_voice(
+        self,
+        audio_file_path: str,
+        voice_id: str,
+        output_format_container: str,
+        output_format_sample_rate: int,
+        output_format_encoding: Optional[str] = None,
+        output_format_bit_rate: Optional[int] = None
+    ) -> str:
+        """Change the voice in an audio file.
+
+        Args:
+            audio_file_path (str): Path to the audio file to change.
+            voice_id (str): The ID of the target voice.
+            output_format_container (str): The format container for the output audio.
+            output_format_sample_rate (int): The sample rate for the output audio.
+            output_format_encoding (Optional[str], optional): The encoding for raw/wav containers. Defaults to None.
+            output_format_bit_rate (Optional[int], optional): The bit rate for mp3 containers. Defaults to None.
+
+        Returns:
+            str: JSON string containing the result information.
+        """
+        logger.info(f"Changing voice in audio file: {audio_file_path}")
+        try:
+            with open(audio_file_path, "rb") as file:
+                params = {
+                    "clip": file,
+                    "voice_id": voice_id,
+                    "output_format_container": output_format_container,
+                    "output_format_sample_rate": output_format_sample_rate
+                }
+
+                if output_format_encoding:
+                    params["output_format_encoding"] = output_format_encoding
+
+                if output_format_bit_rate:
+                    params["output_format_bit_rate"] = output_format_bit_rate
+
+                result = self.client.voice_changer.bytes(**params)
+                return json.dumps({"success": True, "result": str(result)[:100] + "..."}, indent=4)
+        except Exception as e:
+            logger.error(f"Error changing voice with Cartesia: {e}")
+            return json.dumps({"error": str(e)})
+
+    def text_to_speech(
+        self,
+        model_id: str,
```

Comment:
This should be set on initialization

Reply:
But then you can add multiple instances of the toolkit. Having the model decide which model ID to set is weird, because how would it know?